packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -392,7 +392,16 @@ export function TransferPanel() {
           )
         }
         if (latestNetworksAndSigners.current.isConnectedToArbitrum) {
-          trackEvent('Switch Network and Transfer')
+          if (shouldTrackAnalytics(l2NetworkName)) {
+            trackEvent('Switch Network and Transfer', {
+              type: 'Deposit',
+              tokenSymbol: selectedToken?.symbol || undefined,
```

Comment:
Do we need the `|| undefined` here? It would only be defaulted to `undefined` if `selectedToken` is not defined. If we do, then we should have it on L484, L507, L622, L645 too

Reply:
We don't need it, removing